mautic/mautic
# Fix: Tags filter Tag IDs are displayed as dropdown options instead of…

… Tag Name



| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| New feature/enhancement? (use the a.x branch)      | ❌ 
| Deprecations?                          | ❌ 
| BC breaks? (use the c.x branch)        | ❌ 
| Automated tests included?              | ❌  
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



For Value field for Tags filter Tag IDs are displayed as dropdown options instead of Tag Name

#### Steps to reproduce:
1. Create at-least 2 contacts with different tags
2. Create dynamics content without Campaign based flag
3. Try use tags filter - The ids appear in the dropdown

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Try steps to reproduce




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11088"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
